src/trace-api.ts

```diff
@@ -102,6 +103,7 @@ export class StackdriverTracer implements Tracer {
    */
   constructor(name: string) {
     this.pluginName = name;
+    this.pluginNameToLog = this.pluginName ? this.pluginName : 'noPluginName';
```

Comment:
Small nit -- would you mind changing this to `'no-plugin-name'`? Just to keep it consistent, as existing plugin names don't use camelcase.

Reply:
Sounds good. I'll update now.

Reply:
Updated in https://github.com/googleapis/cloud-trace-nodejs/pull/958/commits/5ec2549b2b6c11707ae94f50bd13ff8dcbd906c9